test/util/testnode/utils.go

```diff
@@ -65,10 +65,9 @@ func RandomAddress() sdk.Address {
 	return addresses[0]
 }
 
-func FundKeyringAccounts(accounts ...string) (keyring.Keyring, []banktypes.Balance, []authtypes.GenesisAccount) {
-	kr, addresses := NewKeyring(accounts...)
-	genAccounts := make([]authtypes.GenesisAccount, len(accounts))
-	genBalances := make([]banktypes.Balance, len(accounts))
+func FundKeyringAccounts(addresses []sdk.AccAddress) ([]banktypes.Balance, []authtypes.GenesisAccount) {
```

Comment:
[informational] note this modification is technically Go API breaking. We technically don't need to preserve Go API compatability because this is targeting `main` so FYI but no change needed

Reply:
I would prefer if we add this as a completely new function called `FundAddresses` since it's quite a different function

Reply:
I'll revert this change completely. I thought that decoupling those functions made sense but we don't really use them elsewhere.